lib/Runtime/Library/JavascriptString.cpp

```diff
@@ -2107,15 +2098,6 @@ namespace Js
         JavascriptString* pThis = nullptr;
         GetThisStringArgument(args, scriptContext, L"String.prototype.toUpperCase", &pThis);
 
-        // Fast path for one character strings
```

Comment:
So we are removing fast path for one character string? This might affect some perf benchmarks, right?